format.md

```diff
@@ -217,25 +265,25 @@ Logical representation:
 
 ```
 [
-    [[(0, 0), (0, 1), (0, 2)]],
+    [[{x: 0, y: 0}, {x: 0, y: 1}, {x: 0, y: 2}]],
```

Comment:
@paleolimbot would you be OK with keeping the examples as is here? I personally find the current version a bit easier to read, and the example is mostly to show the additional nesting, regardless of the exact inner representation. We could add a note that for the struct version, all buffers are the same, except the coordinates buffer would be two buffers for x and y.